### PR TITLE
Example input in the manual

### PR DESCRIPTION
**Pull request by swmoon-unist**

### CODE REVIEW

File: docs/source/objects/qm/prog_columbus.rst [hunk 1]
```diff
@@ -245,6 +273,34 @@ MRCI
...
+   example
+   O  1.0 1.0 0.0 0.0 0.0 0.0
+   H  1.0 0.0 0.0 0.0 0.0 0.0
+   H  1.0 1.0 1.0 0.0 0.0 0.0
+   '''
+   
+   mol = Molecule(geometry=geom, ndim=3, nstates=2, unit_pos='au')
+   
+   qm = qm.columbus.MRCI(molecule=mol, basis_set='6-31g*', guess='hf', \
+       state_avg=None, active_elec=2, active_orb=2, frozen_core_orb=0, \
```
Comment (inseonglee) [inline docs/source/objects/qm/prog_columbus.rst:296]: In this example, `frozen_core_orb` must be 1.
If you change the molecule to CH2NH2+, then it becomes 2.

File: docs/source/objects/qm/prog_columbus.rst [hunk 2]
```diff
@@ -245,6 +273,34 @@ MRCI
...
+       frozen_virt_orb=0, qm_path='/opt/columbus7.0/columbus')
```
Comment (inseonglee) [inline docs/source/objects/qm/prog_columbus.rst:297]: change `qm_path` referring upper comment.

File: docs/source/objects/qm/prog_dftbplus.rst [hunk 3]
```diff
@@ -125,6 +125,34 @@ speed. (TD)DFTB and SSR methods are interfaced with the current version of PyUNI
...
+   O  1.0 1.0 0.0 0.0 0.0 0.0
+   H  1.0 0.0 0.0 0.0 0.0 0.0
+   H  1.0 1.0 1.0 0.0 0.0 0.0
+   '''
+   
+   mol = Molecule(geometry=geom, ndim=3, nstates=2, unit_pos='au')
+   
+   qm = qm.dftbplus.DFTB(molecule=mol, l_scc=True, unpaired_elec=0, guess='h0', \
+       ex_symmetry='singlet', sk_path='./', \
+       install_path='/opt/dftbplus-20.1/install-mpi/')
```
Comment (inseonglee) [inline docs/source/objects/qm/prog_dftbplus.rst:149]: In this example, the path should be .../install-openmp/.

File: docs/source/objects/qm/prog_dftbplus.rst [hunk 4]
```diff
@@ -386,6 +414,34 @@ determined from the **l_state_interactions** parameter.
...
+   O  1.0 1.0 0.0 0.0 0.0 0.0
+   H  1.0 0.0 0.0 0.0 0.0 0.0
+   H  1.0 1.0 1.0 0.0 0.0 0.0
+   '''
+   
+   mol = Molecule(geometry=geom, ndim=3, nstates=2, unit_pos='au')
+   
+   qm = qm.dftbplus.SSR(molecule=mol, l_scc=True, l_ssr22=True, guess='h0', \
+       shift=0.3, embedding=None, sk_path='./', \
+       install_path='/opt/dftbplus-20.1/install-mpi/')
```
Comment (inseonglee) [inline docs/source/objects/qm/prog_dftbplus.rst:438]: same comment.

File: docs/source/objects/qm/prog_gaussian09.rst [hunk 5]
```diff
@@ -50,6 +50,33 @@ Gaussian 09 :cite:`Frisch2009` has been a standard program for electronic struct
...
+   example
+   O  1.0 1.0 0.0 0.0 0.0 0.0
+   H  1.0 0.0 0.0 0.0 0.0 0.0
+   H  1.0 1.0 1.0 0.0 0.0 0.0
+   '''
+   
+   mol = Molecule(geometry=geom, ndim=3, nstates=2, unit_pos='au')
+   
+   qm = qm.gaussian09.DFT(molecule=mol, functional='BLYP', basis_set='STO-3g', \
+       guess='Harris', root_path='/opt/gaussian/g09/')
```
Comment (inseonglee) [inline docs/source/objects/qm/prog_gaussian09.rst:73]: Here '/opt/gaussian/g09' should be changed to '/opt/gaussian/'.

File: docs/source/objects/qm/prog_dftbplus.rst [hunk 6]
```diff
@@ -125,6 +125,34 @@ speed. (TD)DFTB and SSR methods are interfaced with the current version of PyUNI
...
+   '''
+   
+   mol = Molecule(geometry=geom, ndim=3, nstates=2, unit_pos='au')
+   
+   qm = qm.dftbplus.DFTB(molecule=mol, l_scc=True, unpaired_elec=0, guess='h0', \
+       ex_symmetry='singlet', sk_path='./', \
+       install_path='/opt/dftbplus-20.1/install-mpi/')
+   
+   md = mqc.SHXF(molecule=mol, nsteps=100, nesteps=20, dt=0.5, unit_dt='au', \
+       sigma=0.1, istate=1, hop_rescale='energy', elec_object='density')
```
Comment (inseonglee) [inline docs/source/objects/qm/prog_dftbplus.rst:152]: Please, add `hop_reject='keep'`.

File: docs/source/objects/qm/prog_gaussian09.rst [hunk 7]
```diff
@@ -50,6 +50,33 @@ Gaussian 09 :cite:`Frisch2009` has been a standard program for electronic struct
...
+   
+   md = mqc.SHXF(molecule=mol, nsteps=100, nesteps=20, dt=0.5, unit_dt='au', \
+       sigma=0.1, istate=1, hop_rescale='energy', elec_object='density')
```
Comment (inseonglee) [inline docs/source/objects/qm/prog_gaussian09.rst:76]: Please, add `hop_reject='keep'`.

File: docs/source/objects/qm/prog_qchem.rst [hunk 8]
```diff
@@ -58,6 +58,33 @@ Q-Chem :cite:`qchem2015` is a comprehensive ab initio quantum chemistry software
...
+   example
+   O  1.0 1.0 0.0 0.0 0.0 0.0
+   H  1.0 0.0 0.0 0.0 0.0 0.0
+   H  1.0 1.0 1.0 0.0 0.0 0.0
+   '''
+   
+   mol = Molecule(geometry=geom, ndim=3, nstates=2, unit_pos='au')
+   
+   qm = qm.qchem.DFT(molecule=mol, functional='blyp', basis_set='sto-3g', \
+       root_path='/opt/qchem/bin/')
```
Comment (inseonglee) [inline docs/source/objects/qm/prog_qchem.rst:81]: `root_path` is '/opt/qchem/'.

File: docs/source/objects/qm/prog_terachem.rst [hunk 9]
```diff
@@ -98,6 +98,33 @@ determined from the **l_state_interactions** parameter.
...
+   example
+   O  1.0 1.0 0.0 0.0 0.0 0.0
+   H  1.0 0.0 0.0 0.0 0.0 0.0
+   H  1.0 1.0 1.0 0.0 0.0 0.0
+   '''
+   
+   mol = Molecule(geometry=geom, ndim=3, nstates=2, unit_pos='au')
+   
+   qm = qm.terachem.SSR(molecule=mol, l_ssr22=True, guess='dft', basis_set='sto-3g'\
+       shift=0.3, install_path='/opt/terachem1.93/TeraChem/bin/')
```
Comment (inseonglee) [inline docs/source/objects/qm/prog_terachem.rst:121]: The parameter name is `qm_path`, not `install_path`.

File: docs/source/objects/qm/prog_turbomole.rst [hunk 10]
```diff
@@ -54,6 +54,34 @@ in the group of Prof. Dr. Reinhart Ahlrichs at the University of Karlsruhe and a
...
+   H  1.0 1.0 1.0 0.0 0.0 0.0
+   '''
+   
+   mol = Molecule(geometry=geom, ndim=3, nstates=2, unit_pos='au')
+   
+   qm = qm.turbomole.DFT(molecule=mol, functional='b-lyp', basis_set='SV(P)', \
+       root_path='/opt/turbomole/bin/')
+   
+   md = mqc.SHXF(molecule=mol, nsteps=100, nesteps=20, dt=0.5, unit_dt='au', \
+       sigma=0.1, istate=1, hop_rescale='energy', elec_object='density')
```
Comment (inseonglee) [inline docs/source/objects/qm/prog_turbomole.rst:80]: In current code, only BOMD is possible with TD-DFT using Turbomole.

File: docs/source/objects/qm/prog_turbomole.rst [hunk 11]
```diff
@@ -54,6 +54,34 @@ in the group of Prof. Dr. Reinhart Ahlrichs at the University of Karlsruhe and a
...
+   example
+   O  1.0 1.0 0.0 0.0 0.0 0.0
+   H  1.0 0.0 0.0 0.0 0.0 0.0
+   H  1.0 1.0 1.0 0.0 0.0 0.0
+   '''
+   
+   mol = Molecule(geometry=geom, ndim=3, nstates=2, unit_pos='au')
+   
+   qm = qm.turbomole.DFT(molecule=mol, functional='b-lyp', basis_set='SV(P)', \
+       root_path='/opt/turbomole/bin/')
```
Comment (inseonglee) [inline docs/source/objects/qm/prog_turbomole.rst:77]: `root_path` is '/opt/TURBOMOLE/'.

File: docs/source/objects/qm/prog_columbus.rst [hunk 12]
```diff
@@ -76,6 +76,33 @@ In the current version of PyUNIxMD, (SA-)CASSCF and MRCI methods are available.
...
+   example
+   O  1.0 1.0 0.0 0.0 0.0 0.0
+   H  1.0 0.0 0.0 0.0 0.0 0.0
+   H  1.0 1.0 1.0 0.0 0.0 0.0
+   '''
+   
+   mol = Molecule(geometry=geom, ndim=3, nstates=2, unit_pos='au')
+   
+   qm = qm.columbus.CASSCF(molecule=mol, basis_set='6-31g*', guess='hf', \
+       state_avg=None, active_elec=2, active_orb=2, qm_path='/opt/columbus7.0/columbus')
```
[the remark this answers is not in the document]
Comment (jkha-unist) [reply docs/source/objects/qm/prog_columbus.rst:99]: Although this is slightly off-topic, at some point, I think it's better to make the code run without the last "/", i.e. automatically add "/" in the code.

[the remark this answers is not in the document]
Comment (inseonglee) [reply docs/source/objects/qm/prog_columbus.rst:99]: Definitely, it is essential part.

File: docs/source/objects/qm/prog_dftbplus.rst [hunk 13]
```diff
@@ -125,6 +125,34 @@ speed. (TD)DFTB and SSR methods are interfaced with the current version of PyUNI
 | *(string)*             |                                                |                    |
 +------------------------+------------------------------------------------+--------------------+
 
+Example input for DFTB+
```
Comment (inseonglee) [inline docs/source/objects/qm/prog_dftbplus.rst:128]: DFTB+ -> (TD)DFTB

**Comment by inseonglee** [comment]
Overall, please remove useless blank lines.

**Pull request merged**